### ISSUE TITLE
Always hidden icons inside the always hidden section is still visible.

### ISSUE THREAD

**Issue by dbenedictos** [dwarvesf/hidden#224]
The icons inside the always hidden section is still visible after expanding.

steps to reproduce:
1. add any icon to the always hidden section.
2. collapse the bar.
4. expand the bar.

app version: 1.8 (12) from mac app store
os: macOS 12.5.1

video for reference:
https://user-images.githubusercontent.com/47196695/188254887-93b32e61-c538-44de-a899-5c092194fc1b.mov

**Comment by PlayingWithPi** [comment]
Hi, I seem to be having this issue as well!

Can this please be looked into?

**Comment by ach881** [comment]
I have the same problem. 

App version 1.9 (13); downloaded from github

MacOS 12.5.1
Macbook Air 2020 M1 8C CPU 8C GPU, 8 GB ram

**Comment by albas99** [comment]
Same issue as well here: https://app.usebubbles.com/vSJnZRX3mF2WWSXfRzz3an/hidden-bar-issue

App version 1.8(App store)
MacOS 12.5.1

**Comment by mlcampbe** [comment]
My findings show that after you enable the "always hidden" option you have to quit hidden and restart it and then move icons to the appropriate section. It should work after that.

**Comment by ach881** [comment]
@mlcampbe Thanks, that gets "always hidden" working for me. Odd behavior!

**Comment by crankygeek** [comment]
mlcampbe's trick isn't working for me on MacBook Pro (16-inch, 2019) macOS 12.5.1.

**Comment by shbhnkr-sg** [comment]
Right-click on the arrow button to toggle between always hidden visible and hidden

**Comment by crankygeek** [comment]
@shbhnkr-sg Thanks! I completely missed the step to right-click in the popover help. I did need to restart the app, but after that, it's working great.

**Comment by Gitbreast** [comment]
> My findings show that after you enable the "always hidden" option you have to quit hidden and restart it and then move icons to the appropriate section. It should work after that.

So weird but work! Seems like a little bug.

**Comment by p3P4** [comment]
Bug Confirmed, Workaround from Gitbreast too.
Version 1.9, Ventura 13.0.1

**Comment by Connor-Kinging** [comment]
> 我的发现表明，启用“始终隐藏”选项后，您必须退出隐藏并重新启动它，然后将图标移动到适当的部分。它应该在那之后工作。

yes，is useful

**Comment by raztronaut** [comment]
> My findings show that after you enable the "always hidden" option you have to quit hidden and restart it and then move icons to the appropriate section. It should work after that.

@mlcampbe's fix works for me!
Using Version 1.9 (13) an old mid-2015 MacBook Pro running on Monterey 12.6.

**Comment by studiohansen** [comment]
this i tried out several times - this unfortounately does not work for me…

after restart hidden bar no single symbol of hidden bar is shown not the arrow, not even one of the vertikal lines…
nothing…

its a mess...



Am 20.01.2023 um 01:58 schrieb Razi Syed ***@***.***>:


My findings show that after you enable the "always hidden" option you have to quit hidden and restart it and then move icons to the appropriate section. It should work after that.

@mlcampbe <https://github.com/mlcampbe>'s fix works for me!
Using Version 1.9 (13) an old mid-2015 MacBook Pro running on Monterey 12.6.

**Comment by raztronaut** [comment]
@studiohansen are you right-clicking the "<"?

**Comment by studiohansen** [comment]
thanks for answering me…


there is no > and no < anymor shown - even no |

i don’t know what happened…


since i tried out the  „always hidden“ version with two |.  
before everything worked well…  f…

i made the mistake and uninstalled hidden bar and hoped, the icons would have come back.
but the never came back with or withoiut hidden bar app

after reinstall hidden bar these control signs aren’t shown anymore



it’s a mess



Am 20.01.2023 um 17:07 schrieb Razi Syed ***@***.***>:


@studiohansen <https://github.com/studiohansen> are you right-clicking the "<"?

**Comment by studiohansen** [comment]
problem solved…

Thank you for the hint with the right mouse button: after I was completely desperate, I wildly right-clicked everywhere in the menu bar. Suddenly an icon appeared that looked familiar: it was the hide menu bar function from the app "Only Switch".
It seems that the two programmes don't get along. Although I had turned off the hide function in Only Switch, it seems that using Hidden Bar simply turns it back on. The two programmes do not get along. The solution is to set the separator symbol for objects to be hidden in Only Switch to the far left. Then press the right mouse button and the symbol disappears. It seems that the two programmes do not get along with each other. Although I had switched off the hide function in Only Switch, it seems to be switched on again by using Hidden Bar. The two programmes do not get along. The solution is to hide the separation symbol for the objects in Only Switch and then press the right mouse button. This hides the symbol and Hidden Bar can then be used normally. Or it is best to do without Only Switch. 
Maybe someone else knows more about this, I can only say I am glad to have my usual working environment back. 

Hidden Bar and Only Switch: Caution when using them at the same time

Many thanks for your help, 
Holger.


Am 20.01.2023 um 17:07 schrieb Razi Syed ***@***.***>:


@studiohansen <https://github.com/studiohansen> are you right-clicking the "<"?

**Comment by Drallas** [comment]
The (right-clicking the "<") should be document somewhere perhaps in readme.md - Usage and the overview App screen!

**Comment by pencilheart** [comment]
restart app works
